src/commands/createFunctionApp/FunctionAppCreateStep.ts

```diff
@@ -136,6 +127,147 @@ export class FunctionAppCreateStep extends AzureWizardExecuteStep<IFunctionAppWi
         return site;
     }
 
+    private async getNewSite(context: IFunctionAppWizardContext, stack: FullFunctionAppStack): Promise<Site> {
```

Comment:
Some of the diff here is showing up as new code, but much of it is just me reorganizing existing code sections and grouping them under foldable regions.  Due to having new methods related to `createSiteWithDomainLabelScope`, it made sense to try and group related logic together.